salt/modules/pillar.py

```diff
@@ -62,6 +63,24 @@ def get(key,
             then merging will be skipped and the retrieved value will be
             returned. Earlier releases raised an error in these cases.
 
+    deep_merge_lists : ``None``
```

Comment:
Here's the scenario that caused me to move away from `merge_lists` in the first place...

If `default` and the returned value are both lists, then `merge=True` _as is_, _without_ this PR, merges the lists and removes duplicates. So, say a user calls pillar.get with `merge=True` and `merge_lists=False`. They are telling us _not_ to merge lists, but the result will be a merged list. Not good.

So, that's the scenario I was trying to avoid by calling the argument `deep_merge_lists`. I'm happy to rename it to something else, but I don't think `merge_lists` is a good option.